worlds/LauncherComponents.py

```diff
@@ -91,7 +91,7 @@ def launch_textclient():
                                                '.apsmw', '.apl2ac')),
     # BizHawk
     Component("BizHawk Client", "BizHawkClient", component_type=Type.CLIENT,
-              file_identifier=SuffixIdentifier()),
+              file_identifier=SuffixIdentifier(".apemerald")),
```

Comment:
wow i really hate this, it isn't apworld friendly at all. would recommend a module level list, that apworlds can inject their extension into. maybe `bizhawk_identifiers` and have it live in `LauncherComponents`.

Reply:
it'd actually probably be better to just move this entire component into the bizhawk apworld and have the identifiers list there as well. obviously since the world can import bizhawk client stuff, it can also access a module level list there. not sure why i thought it wouldn't be able to.

Reply:
This is out of scope for this PR. I'll come up with something, make a PR for it, and link it here.

Reply:
Addressed in #2367. Once that merges I'll make the changes here.

Reply:
#2367 was merged, which removed the lines this convo is attached to. 9411266 adds the suffix.